articles/draas/draas-gs.md

```diff
@@ -80,37 +80,39 @@ You should consider the following:
 
 To use DRaaS, you must have a valid, centrally managed vSphere or HyperV estate. A single hypervisor is not sufficient.
 
-The following table lists the minimum requirements for DRaaS, based on the currently deployed version of Zerto Virtual Replication (4.5U4) on the UKCloud platform:
+The following table lists the requirements for DRaaS, based on the currently deployed version of Zerto Virtual Replication (6.5U3) on the UKCloud platform:
 
 Product | Supported Version
 --------|------------------
 **Management Plane** | &nbsp;
-VMWare vCenter | 4.0U1 -- 6.0U2
-Microsoft SCVMM | 2012 R2 and Server Core: Update Rollup 6-11 (recommended)</br>2012 R2 and Server Core: RTM
+VMWare vCenter | vCenter Server 5.0, 5.1, 5.5, 6.0, 6.5, 6.5U1, 6.5U2, 6.7, 6.7U1
+Microsoft SCVMM | 2012 R2 and Server Core: Update Rollup 1-5, 6-11, 12-14<br>2012 R2 and Server Core: RTM & RTM Rollup 3-5, 6
```

Comment:
Second line should be 2016 (page 21 of IOM):
2016 and Server Core: RTM Rollup 3-5, 6

Reply:
Good spot, thanks. Fixed.